src/azure-cli-core/azure/cli/core/tooling.py

```diff
@@ -0,0 +1,194 @@
+"""tooling integration"""
```

Comment:
And are these code production code or just serves the test purpose? If these are production code, I wonder if you can write your own wrap to hosting these code.